### PR TITLE
Add CARDANO_MAINNET_MIRROR to Nix shell

### PR DESCRIPTION
**Pull request by ana-pantilie**
# Description

# Checklist

- [ ] Commit sequence broadly makes sense and commits have useful messages
- [ ] New tests are added if needed and existing tests are updated
- [ ] When applicable, versions are updated in `.cabal` and `CHANGELOG.md` files according to the
      [versioning process](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#versioning-process).
- [ ] The version bounds in `.cabal` files for all affected packages are updated. **If you change the bounds in a cabal file, that package itself must have a version increase.** (See [RELEASING.md](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#versioning-process))
- [ ] All visible changes are prepended to the latest section of a `CHANGELOG.md` for the affected packages. **New section is never added with the code changes.** (See [RELEASING.md](https://github.com/intersectmbo/cardano-ledger/blob/master/RELEASING.md#changelogmd))
- [x] Code is formatted with [`fourmolu`](https://github.com/fourmolu/fourmolu) (use `scripts/fourmolize.sh`)
- [x] Cabal files are formatted (use `scripts/cabal-format.sh`)
- [x] [`hie.yaml`](https://github.com/intersectmbo/cardano-ledger/blob/master/hie.yaml) has been updated (use `scripts/gen-hie.sh`)
- [ ] Self-reviewed the diff

### CODE REVIEW

**Review by lehins — APPROVED** [review]
LGTM
Thanks

File: README.md
Comment (lehins) [inline README.md:1]: Let's hold of on merging this PR for now. I have another PR in flight that I want to get merged first

**Comment by lehins** [comment]
I'll merge this PR when we are ready.
FYI. We normally do not let merge commits into feature branches and ask to rebase on master instead. There is a feature in the UI for this as well "Update with rebase":

image

**Comment by ana-pantilie** [comment]
@lehins Sorry, thank you for fixing it! I'll make sure to remember this for future PRs.

**Comment by lehins** [comment]
@ana-pantilie No worries, you had no way of knowing. I only pointed it out for the future.
Thank you for submitting this PR.

**Pull request merged**